--- BitchX/include/config.h.orig	Sun Mar 24 10:31:04 2002
+++ BitchX/include/config.h	Mon May 13 03:05:00 2002
@@ -37,7 +37,9 @@
  *
  * List last updated: 03/24/01.  -dialtone.
  */
-#define DEFAULT_SERVER  "[efnet] "\
+#define DEFAULT_SERVER  "[PLD] "\
+				"irc.pld.org.pl "\
+			"[efnet] "\
 				"irc.Prison.NET "\
 				"irc.colorado.edu "\
 				"irc.concentric.net "\
@@ -69,6 +71,11 @@
 				"efnet.mit.edu "\
 				"efnet.vuurwerk.nl "\
 			"[ircnet] "\
+				"poznan.irc.pl" \
+				"wroclaw.irc.pl" \
+				"krakow.irc.pl" \
+				"lublin.irc.pl" \
+				"warszawa.irc.pl" \
 				"irc.webbernet.net "\
 				"irc.stealth.net "\
 				"irc.funet.fi "\
@@ -181,7 +188,7 @@
 #if defined(WINNT) || defined(__EMX__)
 #define SERVERS_FILE "irc-serv"
 #else
-#define SERVERS_FILE "ircII.servers"
+#define SERVERS_FILE "/etc/irc/ircII.servers"
 #endif
 
 /*
@@ -195,7 +202,7 @@
  * we define the default network type for server groups. Do not just
  * undefine this.
  */
- #define DEFAULT_NETWORK "efnet"
+ #define DEFAULT_NETWORK "PLD"
 
 /*
  * Below are the IRCII variable defaults.  For boolean variables, use 1 for
@@ -333,7 +340,7 @@
  * define this to solve the problem.
  *
  */
-#undef CHARSET_CUSTOM
+#define CHARSET_CUSTOM
 
 /* 
  * If you want the non-ansi BitchX logo only define this ASCII_LOGO
