diff -ru BitchX/Makefile.in BitchX.AX/Makefile.in
--- BitchX/Makefile.in	Fri Nov 19 19:15:08 1999
+++ BitchX.AX/Makefile.in	Thu Jan 20 20:14:47 2000
@@ -42,7 +42,7 @@
 # Where the BitchX library will be.  Generally this is the place that
 # you put the scripts, help pages and translation tables.  It is
 # very important that you set this correctly.
-IRCLIB = @IRCLIB@
+IRCLIB = ${prefix}/share/BitchX
 
 CC = @CC@
 DEFS = -I../include
@@ -97,10 +97,10 @@
 
 # This program allows you to use screen/xterm's to put new BitchX windows
 # on new screen/xterm windows.
-INSTALL_WSERV = @INSTALL_WSERV@
+INSTALL_WSERV = $(IRCLIB)/wserv
 
 # This program allows you to screen BitchX and reattach to it later.
-INSTALL_SCRBX = @INSTALL_SCRBX@
+INSTALL_SCRBX = $(IRCLIB)/scr-bx
 
 ## You shouldn't have to change anything below here
 
diff -ru BitchX/source/translat.c BitchX.AX/source/translat.c
--- BitchX/source/translat.c	Sun Oct 31 12:41:56 1999
+++ BitchX.AX/source/translat.c	Thu Jan 20 20:16:26 2000
@@ -36,7 +36,7 @@
 /* we have this here as a safety feature if for some reason TRANSLATION_PATH
  * is not defined.
  */
-#define TRANSLATION_PATH "/usr/local/lib/bx/translation"
+#define TRANSLATION_PATH "/usr/share/BitchX/translation"
 #endif
 
 
