diff -ru BitchX/source/output.c BitchX.AX/source/output.c
--- BitchX/source/output.c	Tue Jan  4 08:27:26 2000
+++ BitchX.AX/source/output.c	Sat Jan 29 23:41:06 2000
@@ -49,17 +49,17 @@
 
 void charset_ibmpc (void)
 {
-	fwrite("\033(U", 3, 1, current_ftarget);	/* switch to IBM code page 437 */
+//	fwrite("\033(U", 3, 1, current_ftarget);	/* switch to IBM code page 437 */
 }
 
 void charset_lat1 (void)
 {
-	fwrite("\033(B", 3, 1, current_ftarget);	/* switch to Latin-1 (ISO 8859-1) */
+//	fwrite("\033(B", 3, 1, current_ftarget);	/* switch to Latin-1 (ISO 8859-1) */
 }
 
 void charset_cst(void)
 {
-	fwrite("\033(K", 3, 1, current_ftarget); /* switch too user-defined */
+//	fwrite("\033(K", 3, 1, current_ftarget); /* switch too user-defined */
 }
 
 /* currently not used. */
